src/coreclr/tools/Common/Internal/Text/Utf8StringBuilder.cs

```diff
@@ -99,53 +89,8 @@ private void Grow(int extraSpace)
         {
             int newSize = Math.Max(2 * _buffer.Length, _length + extraSpace);
             byte[] newBuffer = new byte[newSize];
-            Buffer.BlockCopy(_buffer, 0, newBuffer, 0, _length);
+            _buffer.AsSpan(0, _length).CopyTo(newBuffer);
```

Comment:
```suggestion
            AsSpan().CopyTo(newBuffer);
```

Reply:
This suggestion will copy more than strictly required into the new buffer. It won't hurt, but it is unlikely to be an improvement.